### ISSUE TITLE
dropout not available in the pretrained models 

### ISSUE THREAD

**Issue by mojtababahrami1993** [tencent-ailab/grover#8]
When using the pre-trained models (both grover_base.pt and grover_large.pg) and trying to predict fingerprints for `bbbp.csv` example dataset the following error is raised saying that the `args.dropout` field is not found in the `state['args']`:

![image](https://user-images.githubusercontent.com/58011118/128597266-a6d74c4e-bdf7-490f-8c97-7ce5581157a2.png)

**Comment by yuliang0225** [comment]
I found this Bug. Therefore, I added one line "args.dropout = 1" at 36 line in grover/model/models.py.
